exercises/anagram/README.md

```diff
@@ -6,6 +6,23 @@ Given a word and a list of candidates, select the sublist of anagrams of the giv
 Given `"listen"` and a list of candidates like `"enlists" "google"
 "inlets" "banana"` the program should return a list containing
 `"inlets"`.
+
+# Running and testing your solutions
+
+## Overview
+
+Suppose you're solving __hello\-world__:
+
+* Start a REPL, either in your favorite editor or from the
+command line\.
+* Type `(load "hello-world.scm")` at the prompt\.
+* Test your code by calling `(test)` from the REPL\.
+
+## Testing options
+
+You can see more information about failing test cases by passing
+arguments to the procedure `test`\. 
+To see the failing input call `(test 'input)` and to see the output as well call `(test 'input 'output)`\.
 ## Source
```

Comment:
Should there be a newline above this?  Not every track has one here.
It might look more consistent. I could go either way.

(obviously the question is for a change to all of the readme's)

Reply:
Nevermind.  Markdown is smart enough to make it look good on its own.